pvlib/spectrum/mismatch.py

```diff
@@ -0,0 +1,164 @@
+"""
+The ``mismatch`` module provides functions for spectral mismatch calculations.
+"""
+
+import pvlib
+import numpy as np
+import pandas as pd
+from scipy.interpolate import interp1d
+import os
+
+
+def get_sample_sr(wavelength=None):
+    '''
+    Generate a generic smooth c-si spectral response for tests and experiments.
+
+    Parameters
+    ----------
+    wavelength: 1-D sequence of numeric, optional
+        The wavelengths at which the sample sr should be interpolated. [nm]
+        By default the wavelengths are from 280 to 1200 in 5 nm intervals.
+
+    Notes
+    -----
+    This sr is based on measurements taken on a reference cell.
+    The measured data points have been adjusted by PV Performance Labs so that
+    standard cubic spline interpolation produces a curve without oscillations.
+    '''
+    SR_DATA = np.array([[ 290, 0.00],
```

Comment:
Maybe name "SR_SAMPLE_DATA"

Reply:
I think it's fine as an internal name in the function.